graphene_django/views.py

```diff
@@ -186,7 +186,7 @@ def dispatch(self, request, *args, **kwargs):
                     or 200
                 )
             else:
-                result, status_code = self.get_response(request, data, show_graphiql)
+                result, status_code = self.get_response(request, data)
```

Comment:
The preceding code does not allow `show_graphiql` to ever be `True` here:
https://github.com/graphql-python/graphene-django/blob/10793befb0586543d2b4965935d8c2c2877f1c50/graphene_django/views.py#L153-L156
Ergo, there is no reason to pass it around or pivot any other behavior on whether `show_graphiql` is `True` or not so I have removed this pivot to simplify downstream logic.